shopinvader_api_cart/security/groups.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2023 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+  <record id="sale_cart_user_group" model="res.groups">
+    <field name="name">Sale Cart user</field>
```

Comment:
I think that you should add  
```
<field
      name="implied_ids"
      eval="[
      (4, ref('fastapi.group_fastapi_endpoint_runner')),
      ]"
    />
```

Reply:
https://github.com/OCA/rest-framework/blob/c63e21d0e62a503ad78efc3af06675ec425ed05e/fastapi/readme/USAGE.rst?plain=1#L200

Reply:
Done in the last commit.